src/pages/GoverningBody/Sector/AddSectorAdminForm.tsx

```diff
@@ -159,15 +147,22 @@ const AddSectorAdminForm = (props: any) => {
   useEffect(() => {
     const fetchData = async () => {
       setUsersLoading(true);
-      await adminApi.getUsersForTable().then((response) => {
-        setUsers(response.data);
+      await adminApi.getUsersByAnyRole([
+        Roles.RegisteredUser, 
+        Roles.Supporter, 
+        Roles.FormerPlastMember, 
+        Roles.Interested,
+        Roles.GoverningBodySectorHead, 
+        Roles.GoverningBodySectorSecretary],
+        false)
+      .then((response) => {
+        setUsers(response.data)
         setUsersLoading(false);
       });
```

Comment:
Should not be also catch block in case of any error, or it is overkill?
```suggestion
      }).catch((error) => {
        notificationLogic("error", error.message);
      });
```